src/templates/migration.ts.ejs

```diff
@@ -23,7 +23,17 @@
     // await consul.key('Test_Key').jsonpath('$.b.c').val('test value').save()
     // 
     // results -----> {"a": {"b": {"c": "test value"}}}
-    // 
+    //
+    // Note: use jsonpath to add new key-value pair in consul objects
+    // await consul
+    //  .key(`Test_Key`)
+    //  .jsonpath(`$.b.f`)
+    //  .val({
+    //    "path": "www.test.com",
+    //    "method": "GET"
+    //  })
+    //  .save();
+    //
```

Comment:
Make a note here about this not supporting the full jsonpath spec. This functionality only supports the "." notation from jsonpath. see here for full: https://www.npmjs.com/package//jsonpath

Reply:
added the comment in .ejs file